create-projects/Jenkinsfile

```diff
@@ -67,8 +70,15 @@ podTemplate(
       sh "sh ./create-projects/create-projects.sh"
     }
 
-    stage('creating jenkins') {
-      sh "sh ./create-projects/create-cd-jenkins.sh"
+    // read json reult from Nexus curl call
+    def nexusResponse = readJSON text: sh(script: "curl -sf --user $nexusUser:$nexusPass -X POST -H 'Accept: application/json' -H 'Content-Type: text/plain' \"$nexusHost/service/rest/v1/script/createProjectAccess/run\" -d $projectId", returnStdout: true).trim()
+
+    def specificUser = readJSON text: nexusResponse.result
```

Comment:
Why `readJSON` twice? Shouldn't nexusResponse be a JSON object already?

Reply:
I had the same assumption, but unfortunately it isn't. After reading the nexusResponse into JSON the result field is still a string and has to be converted to an JSON object for itself.